src/raw/mod.rs

```diff
@@ -874,6 +920,55 @@ impl<T> Iterator for RawIter<T> {
         }
     }
 
+    #[inline]
+    fn size_hint(&self) -> (usize, Option<usize>) {
+        // We don't have an item count, so just guess based on the range size.
+        (0, Some(unsafe { offset_from(self.end, self.ctrl) }))
+    }
+}
+
+impl<T> ExactSizeIterator for RawIterRange<T> {}
+impl<T> FusedIterator for RawIterRange<T> {}
+
+/// Iterator which returns a raw pointer to every full bucket in the table.
+pub struct RawIter<T> {
+    pub iter: RawIterRange<T>,
+    items: usize,
+}
+
+unsafe impl<T> Send for RawIter<T> where T: Send {}
+unsafe impl<T> Sync for RawIter<T> where T: Sync {}
```

Comment:
I think these can actually be removed since they are implied by the impl on `RawIterRange`.